examples/jsm/Addons.js

```diff
@@ -71,7 +71,6 @@ export * from './interactive/InteractiveGroup.js';
 export * from './interactive/SelectionBox.js';
 export * from './interactive/SelectionHelper.js';
 
-export { default as IESSpotLight } from './lights/IESSpotLight.js';
```

Comment:
Do you mind adding `RectAreaLightTexturesLib`? It seems this new module is missing...